migration-requirements.txt

```diff
@@ -0,0 +1 @@
+-e git+ssh://git@github.com/Ensembl/ensembl-dj-metadata-orm.git#egg=metadata_orm
```

Comment:
Suggestion: you can specify a destination folder when installing in editable mode (`-e`) from a VCS, so `metadata_orm` package will be easier to find for editing. See: [pip-install-vcs-support](https://pip-python3.readthedocs.io/en/latest/reference/pip_install.html#vcs-support)